acceptancetests/assess_resolve.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+""" Ensure resolve command resolves hook issues.
+
+This test covers the resolve command (and the option --no-retry)
+
+The test:
+  - Uses a simple custom charm that instruments failure in the install hook
+  - Ensures that 'resolve' retries the failed install hook and that the
+    followup 'start' hook is fired.
+  - Ensures that 'resolve --no-retry' does NOT re-run the failed install hook
+    and instead goes ahead and runs the follow up 'started' hook.
+"""
+
+from __future__ import print_function
+
+import argparse
+import logging
+import sys
+
+from deploy_stack import (
+    BootstrapManager,
+    )
+from jujupy.models import (
+    temporary_model,
+)
```

Comment:
there's a closing bracket alignment inconsistency here (see above import)
not sure that pylint says is correct